src/main/java/org/broadinstitute/consent/http/db/mapper/DacWithDatasetsReducer.java

```diff
@@ -58,7 +60,7 @@ public void accumulate(Map<Integer, Dac> container, RowView rowView) {
 
         if (Objects.nonNull(rowView.getColumn("dataset_data_use", String.class))) {
           String duStr = rowView.getColumn("dataset_data_use", String.class);
```

Comment:
Note that this form makes two calls to `rowView.getColumn()`. Storing the result of the first call would avoid this extra fetch.
```suggestion
        String duStr = rowView.getColumn("dataset_data_use", String.class);
        if (duStr != null) {
```